### DISCUSSION TITLE
The Feed is too small in width

### DISCUSSION THREAD

**Discussion by d1ceward** [community/community#54526] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello everyone,

I would like to provide some feedback regarding the layout of the feeds on the desktop version of this platform. Specifically, I find the width of the feed to be too small.

When using the platform on my desktop computer, I often find myself scrolling continuously to view the content in the feed. This makes the user experience less enjoyable and can be frustrating at times. I understand that the feed width may be optimized for mobile devices, but it seems to be a bit narrow for desktop use.

I suggest that Github consider widening the feed on the desktop version to allow for a better user experience. This change would make it easier to read and navigate through the feed, which would ultimately improve the platform's overall usability.

Thank you for taking the time to read my feedback. I hope it can be taken into consideration and potentially implemented in future updates.

**Comment by Kaier33** [comment]
I wrote a Tampermonkey script, to make Feed look easier to read. (interim solution)


// ==UserScript==
// @name         github style
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  github style!
// @author       you
// @match        https://github.com
// @icon         data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==
// @grant        none
// ==/UserScript==

(function() {
    'use strict';
    document.head.insertAdjacentHTML("beforeend", "<style>.d-flex.flex-auto.flex-column.feed-main{max-width: 100%;} .feed-content{margin-left: 120px;} .feed-left-sidebar{width: 300px;} .feed-right-sidebar{margin-left: 30px;}</style>");
})();

**Comment by f-person** [comment]
Thank you! 
Modified `match` for Violentmonkey users:
```js
// @match http*://github.com/
```
